conda/cli/main.py

```diff
@@ -80,12 +80,24 @@ def generate_parser():
     return p, sub_parsers
 
 
+def init_loggers(context):
+    from ..gateways.logging import set_all_logger_level, set_verbosity
```

Comment:
I've been saving the big docstring effort for our API-level stuff.  But I'm not opposed.  What would you make this docstring here?  Just as an example (in numpy format)...